src/cache.cpp

```diff
@@ -150,7 +150,7 @@ namespace {
 		int min_height, max_height;
 		std::function<BitmapRef()> dummy_renderer;
 	} const spec[] = {
-		{ "Backdrop", false, 320, 320, 160, 160, backdrop_dummy_func},
+		{ "Backdrop", false, 320, 320, 160, Player::IsRPG2k() ? 160 : 240, backdrop_dummy_func},
```

Comment:
I would agree on changing the value in general to "240" until the Cache system is refactored. This is wrong for 2k but gets rid of 2k3 debug messages.